### ISSUE TITLE
remote validate passed but submit fail

### ISSUE THREAD

**Issue by xiaofanku** [guillaumepotier/Parsley.js#1331]
`
            <form id="register" method="post" action="${BASE}/member/register" class="theme-form parsley-form" data-parsley-validate>
                <div class="form-group">
                    <label for="names">帐号:</label>
                    <small class="form-text text-muted">名称长度必须介于5个字符和20个字符之间，并且只能使用字母数字字符</small>
                    <input type="text" name="names" size="25" value="${form.names}" class="form-control" placeholder="会员的登录帐号名称" data-parsley-required data-parsley-length="[5, 20]" data-parsley-remote="${BASE}/member/detection/unique" data-parsley-remote-message="用户名已经存在" tabindex="1"/>
                </div>
                <input type="submit" name="register" id="submit" value="免费注册" data-submit-disabled="请稍后" class="btn btn-primary" tabindex="7"/>
            </form>
`

![批注 2020-04-29 185844](https://user-images.githubusercontent.com/324047/80621030-4e722200-8a79-11ea-9f89-3d647516269a.jpg)

i used version: 2.9.1, jquery: 3.2.1, server response: 200. remove data-parsley-remote is worked

**Comment by marcandre** [comment]
I suspect you are using different domains and you have a CORS issue, please see #1039.
If not you will have to produce a minimal reproducible example.

**Comment by xiaofanku** [comment]
> I suspect you are using different domains and you have a CORS issue, please see #1039.
> If not you will have to produce a minimal reproducible example.

not cors, is same domain request.
![批注 2020-04-30 103623](https://user-images.githubusercontent.com/324047/80666325-7770d200-8ace-11ea-9fde-b6d8af39b398.jpg)

**Comment by xiaofanku** [comment]
a public domain test. you see is
![批注 2020-04-30 110035](https://user-images.githubusercontent.com/324047/80667568-19de8480-8ad2-11ea-9aa3-a9e2b69e4bb9.jpg)
 cros

**Comment by xiaofanku** [comment]
i try replace `input type=submit` to `button type=submit` is worked, what is reason?

**Comment by marcandre** [comment]
> i try replace input type=submit to button type=submit is worked, what is reason?

Strange. That should not make any difference. Are you sure that's the only thing you have changed?

**Comment by marcandre** [comment]
I doubled checked that on our remote example it seems to work the same for `<input>` or `<button>`. You can play with it by clicking the codepen button in http://parsleyjs.org/doc/examples/ajax.html for example

**Comment by xiaofanku** [comment]
> > i try replace input type=submit to button type=submit is worked, what is reason?
> 
> Strange. That should not make any difference. Are you sure that's the only thing you have changed?

yes, I'm the only one of the developers

**Comment by xiaofanku** [comment]
> I doubled checked that on our remote example it seems to work the same for `<input>` or `<button>`. You can play with it by clicking the codepen button in http://parsleyjs.org/doc/examples/ajax.html for example

At least I'm sure this is not the cause of CORS，
